nodemon.config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "verbose": true,
+  "ext": "ts",
+  "watch": "packages",
+  "ignore": [
+    "**/__generated__/**",
+    "**/*.js",
+    "**/*.mjs",
+    "**/dist/**",
+    "**/Forc.lock",
+    "**/getSupportedVersions.ts",
```

Comment:
Why do we want to ignore this file when watching for changes?

Reply:
@Dhaiwat10 We must ignore generated/compiled files to avoid recursively triggering file changes.